src/dune_rules/odoc.ml

```diff
@@ -353,6 +351,10 @@ let setup_html sctx (odoc_file : odoc_artefact) =
       "html-generate" ~flags_for:None
       [ A "-o"
       ; Path (Path.build (Paths.html_root ctx))
+      ; A "--support-uri"
+      ; A "_odoc_support"
+      ; A "--theme-uri"
+      ; A "_odoc_support"
```

Comment:
Does this mean we're dropping support for old versions of odoc? We should add that to the `conflicts` section then.

Reply:
No, odoc has supported those arguments since 2.0.0, so that's unchanged. Admittedly I haven't tested it yet though!